### PR TITLE
PLAT-100710: Renaming in VirtualList and Scroller

### PR DESCRIPTION
**Pull request by MikyungKim**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [ ] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
We had some ambiguous variable names. In this PR, we'd like to rename them.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Renamed
innerScrollContainer to scrollInnerContainer
css.content to css.scrollContentWrapper
uiChildContainerRef to scrollContentRef
uiChildAdapter to scrollContentHandle
uiScrollAdapter to scrollContainerHandle
decorateChildProps to assginProperties

Extracted Scrollable from useScroll
Split Scroller to Scroller and ScrollerBasic

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-100710

### CODE REVIEW

**Comment by ybsung** [comment]
The default value for the `type` should be `Native`.
```
static defaultProps = {
...
type: 'JS',
...
}
```

File: packages/ui/Scrollable/Scrollable.js [hunk 1]
```diff
@@ -465,13 +465,13 @@ const useScrollBase = (props) => {
 			noScrollByWheel,
 			overscrollEffectOn,
 			rtl,
-			setUiChildAdapter,
-			setUiScrollAdapter,
+			setScrollContentHandle,
+			setScrollContainerHandle,
```
Comment (ybsung) [inline packages/ui/Scrollable/Scrollable.js]: Please check the order.

**Comment by ybsung** [comment]
```
scrollClasses = classNames(css.scroll, className);
```
How about `scrollContainerClasses` instead of the `scrollClasses`?

**Comment by ybsung** [comment]
```
const assignPropertiesOf = (instance) => (name, properties) => {
```
How about the `collectionOfProperties` instead of `instance`?

**Comment by ybsung** [comment]
scrollContentWrapper => `scrollContentWrapperComponent`?

**Comment by ybsung** [comment]
```
const useSpottable = (props, instances) => {
	const {scrollContentHandle, scrollContentRef} = instances;
```
What is the proper naming for the `instances` for handles and refs?

**Comment by ybsung** [comment]
Let's consider additional naming later. Now LGTM. We need to update theme libraries together.

**Comment by MikyungKim** [comment]
LGTM

**Pull request merged**